sos/plugins/samba.py

```diff
@@ -27,7 +27,11 @@ def setup(self):
             "/etc/samba/smb.conf",
             "/etc/samba/lmhosts",
             "/var/log/samba/log.smbd",
-            "/var/log/samba/log.nmbd"
+            "/var/log/samba/log.nmbd",
+            "/var/log/samba/log.winbindd",
+            "/var/log/samba/log.winbindd-idmap",
+            "/var/log/samba/log.winbindd-dc-connet",
+            "/var/log/samba/log.wb-*"
```

Comment:
@bmr-cymru I can do that.
The limit is in MB, right?
But for --all-logs, I would use no limit -- OK?